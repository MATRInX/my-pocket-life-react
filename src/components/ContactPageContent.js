import React from 'react';

const ContactPageContent = () => (
  <div>
    ContactPageContent content...
  </div>
);

export default ContactPageContent;