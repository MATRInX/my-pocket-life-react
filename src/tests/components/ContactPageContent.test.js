import React from 'react';
import { shallow } from 'enzyme';
import ContactPageContent from '../../components/ContactPageContent';

test('should correctly render ContactPageContent', () => {
  const wrapper = shallow(<ContactPageContent />);
  expect(wrapper).toMatchSnapshot();
});