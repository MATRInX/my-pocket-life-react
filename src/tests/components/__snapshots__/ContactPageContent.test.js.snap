// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render ContactPageContent 1`] = `
<div>
  ContactPageContent content...
</div>
`;
